types/viz.js/full.render.js.d.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference path="common.d.ts" />
```

Comment:
Do these files need to exist? Can you consolidate into one file to reflect the package structure, or at least just use modules so that the declarations within those files don't leak into the global scope?

Reply:
When I had it as a single file, the DefinitelyTyped linter complained because I was declaring modules, so I split it out. So, I guess I'm not sure how to do what you're requesting and still keep the linter happy.

```
declare module 'viz.js' {
  export = Viz;
}

declare module 'viz.js/full.render.js' {
  export const Module: IModule;
  export function render(instance: IModule, src: string, options: IOptions): string;
}
```